includes/blocks/class-kadence-blocks-row-layout-block.php

```diff
@@ -1266,6 +1266,9 @@ public function build_html( $attributes, $unique_id, $content, $block_instance )
 			if ( ! empty( $attributes['anchor'] ) ) {
 				$wrapper_args['id'] = $attributes['anchor'];
 			}
+			if ( ! empty( $attributes['kadenceDynamic'] ) || ! empty( $attributes['kadenceDynamic'] ) || ! empty( $attributes['kadenceDynamic']['bgImg'] ) ) {
```

Comment:
The logic here doesn't make sense to me. What are we checking for here?

Reply:
checking to see if the bg img is dynamic. If it is it needs to be applied directly to the element style attribute so that it overrides the bg img set via css. For the case where the dynamic image is in a post loop.

We could have all bg imgs set this way and cut out the css entirely, but I think the image being dynamic is a smaller case and less efficient than writting it into the style sheet so added this condition in here.

Reply:
removed, will resolve this issue in pro